### PR TITLE
[MS] Update invoice translations

### LINKED ISSUE #8731: [Customer area] Dashboard's last invoices table problems
- [ ] Icon and text vertical alignment problem
- [ ] In french, status and date values are in english

Image

### LINKED ISSUE #8732: [Customer area] Invoices page table problems
- [ ] Icon and text vertical alignment problem
- [ ] In french, status title and value are in english

Image

### PR DESCRIPTION
**Pull request by fabienSvtr**
## Describe your changes

Translation updated:
- Date
- Status

image

## Checklist

Before you submit this pull request, please make sure to:

- [ ] Keep changes in the pull request as small as possible
   
- [ ] Ensure the commit history is sanitized
- [ ] Give a meaningful title to your PR
- [ ] Describe your changes
- [ ] Link any related issue in the description
  
closes #8731
closes #8732

### CODE REVIEW

File: client/src/services/translation.ts [hunk 1]
```diff
@@ -14,6 +15,22 @@ export function getProfileTranslationKey(profile: UserProfile): Translatable {
   return '';
 }
 
+export function getInvoiceStatusTranslationKey(status: InvoiceStatus): Translatable {
+  console.log('status', InvoiceStatus);
```
Comment (fabienSvtr) [inline client/src/services/translation.ts:19]: it will be removed

Comment (NicoTuxx) [reply client/src/services/translation.ts:19]: when?

File: client/src/services/translation.ts [hunk 2]
```diff
@@ -14,6 +15,22 @@ export function getProfileTranslationKey(profile: UserProfile): Translatable {
...
+  switch (status) {
+    case InvoiceStatus.Paid:
+      return 'clientArea.invoices.status.paid';
+    case InvoiceStatus.Draft:
+      return 'clientArea.invoices.status.draft';
+    case InvoiceStatus.Open:
+      return 'clientArea.invoices.status.ready';
+    case InvoiceStatus.Uncollectible:
+      return 'clientArea.invoices.status.Uncollectible';
```
Comment (NicoTuxx) [inline client/src/services/translation.ts:28]: why a capitale letter here and not the other status?

**Review by NicoTuxx — CHANGES_REQUESTED** [review]
The PR is not fixing every problem described in the linked issues: the icon alignment bug is still present.

image
image

File: client/src/locales/fr-FR.json [hunk 3]
```diff
@@ -1513,6 +1513,13 @@
                 "status": "Status",
                 "download": "Télécharger"
             },
+            "status": {
+                "paid": "Payée",
+                "ready": "À payer",
+                "draft": "Brouillon",
+                "uncollectible": "Non trouvée",
+                "void": "Inconnu"
```
Comment (Max-7) [inline client/src/locales/fr-FR.json:1521]: `uncollectible` doesn't mean `not found`, it means that we can't enforce the payment. `void` doesn't mean `unknown`, it means that it's not legally binding.

You can find an explanation for every status on the Stripe website https://docs.stripe.com/invoicing/overview.

File: client/src/locales/fr-FR.json [hunk 4]
```diff
@@ -1513,6 +1513,13 @@
                 "status": "Status",
                 "download": "Télécharger"
             },
+            "status": {
+                "paid": "Payée",
+                "ready": "À payer",
```
Comment (Max-7) [inline client/src/locales/fr-FR.json:1518]: English has `toPay`, French has `ready`.

File: client/src/services/translation.ts [hunk 5]
```diff
@@ -1,6 +1,7 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
 import { InvitationStatus, UserProfile, WorkspaceRole } from '@/parsec';
+import { InvoiceStatus } from '@/services/bms/types';
```
Comment (Max-7) [inline client/src/services/translation.ts:4]: Import it from `@/services/bms`

**Review by Max-7 — APPROVED** [review]

**Comment by fabienSvtr** [comment]
> The PR is not fixing every problem described in the linked issues: the icon alignment bug is still present.
> 
> image image

As discussed, I replaced the download icon, now included in megashark-lib.

**Review by NicoTuxx — APPROVED** [review]

**Pull request merged**